models/classes/LtiProvider/LtiProviderFactory.php

```diff
@@ -97,11 +97,13 @@ public function createFromArray(array $provider): LtiProvider
 
     private function getLtiVersion(array $propertiesValues): string
     {
-        if (empty($propertiesValues[RdfLtiProviderRepository::LTI_VERSION])) {
+        $ltiVersionResource = reset($propertiesValues[RdfLtiProviderRepository::LTI_VERSION]);
+
+        if (!$ltiVersionResource || !$ltiVersionResource instanceof core_kernel_classes_Resource) {
             return '1.1';
         }
 
-        $version = (string)reset($propertiesValues[RdfLtiProviderRepository::LTI_VERSION])->getUri();
+        $version = (string) $ltiVersionResource->getUri();
```

Comment:
If it is a literal you should return the literal value, not fixed 1.1. You have to check if is literal too and return is string value.

Reply:
On playground this is output of ```reset($propertiesValues[RdfLtiProviderRepository::LTI_VERSION])```

```core_kernel_classes_Literal::__set_state(array(
   'literal' => ' ',
))```

Reply:
So maybe you can verify both cases if is literal and empty, then you return 1.1, so we are in the safe side?